src/langEN.js

```diff
@@ -55,5 +55,37 @@ export default {
             title: 'Save as Image',
             lang: ['Right Click to Save Image']
         }
+    },
+    aria: {
+        general: {
+            withTitle: 'This is a chart about "{title}"',
+            withoutTitle: 'This is a chart，'
+        },
+        series: {
+            single: {
+                prefix: '',
+                withName: 'This is a chart with type {seriesType} named {seriesName}.',
```

Comment:
This comes after `general.withTitle` or `general.withoutTitle`, so it should be something like `, and this is ...`.